app/models/bot/alegre.rb

```diff
@@ -364,23 +339,31 @@ def self.split_text(text)
     text.split(/\s/)
   end
 
-  def self.get_items_with_similar_title(pm, threshold, text_length_threshold=self.similarity_text_length_threshold)
-    self.split_text(pm.title.to_s).length > text_length_threshold ? self.get_merged_similar_items(pm, threshold, ['original_title', 'analysis_title'], pm.title) : {}
+  def self.get_items_with_similar_title(pm, threshold)
```

Comment:
Is this part of this ticket to remove the idea of a `similarity_text_length_threshold`? If so, this is fine, otherwise, this is definitely breaking that old logic

Reply:
yes @DGaffney I removed  `similarity_text_length_threshold` condition as short text will be added as suggestion